CHANGELOG.next.md

```diff
@@ -26,6 +26,7 @@ Thanks, you're awesome :-) -->
 
 * Expanded field set definitions for `source.*` and `destination.*`. #967
 * Provided better guidance for mapping network events. #969
+* Resolved inconsistency of domain name breakdown fields. #981
```

Comment:
I'd be more specific here. The following doesn't roll off the tongue as much, but users reviewing the changelog won't need to look up the PR to understand what the change is:

```suggestion
* Added the field `.subdomain` under `client`, `destination`, `server`, `source` and `url`, to match its presence at `dns.question.subdomain`. #981
```